docs/units-and-global-variables.rst

```diff
@@ -136,6 +136,13 @@ ABI Encoding and Decoding Functions
 See the documentation about the :ref:`ABI <ABI>` and the
 :ref:`tightly packed encoding <abi_packed_mode>` for details about the encoding.
 
+.. index:: bytes members
+
+Members of bytes
+----------------
+
+- ``bytes.concat(...) returns (bytes memory)``: :ref:`Concatenates variable number of arguments to one array<bytes-concat>`
```

Comment:
I wasn't sure if I could fit this to any other category, so added a new one.
If there is a better option, please suggest it? :)

Reply:
Looks good!

Reply:
Maybe change to `variable number of bytes and bytes1, ..., bytes32 arguments to one byte array`